package.json

```diff
@@ -18,23 +18,19 @@
   },
   "homepage": "https://github.com/Oli8/spaper#readme",
   "scripts": {
-    "dev": "svelte-kit dev",
```

Comment:
Those scripts are gone?

Reply:
I think they're replaced with vite commands in newer versions of SvelteKit (e.g. `vite dev`).

But my suggestion was to remove SvelteKit entirely. I explain more in the PR description but if you remove it then you can have this as a lib that supports both Svelte 3 and 4 as a peer dependency. If you have SvelteKit as a dependency it'll only be able to support one or the other due to dependency conflicts.